### PR TITLE
Android: Remove GPU driver from per-game settings

### PR DESCRIPTION
**Pull request by K0bin**
Don't allow changing it per game because we can only install one driver. Also don't allow changing it while in-game.

### CODE REVIEW

**Comment by K0bin** [comment]
cc @t895

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/model/StringSetting.kt [hunk 1]
```diff
@@ -101,7 +101,8 @@ enum class StringSetting(
     companion object {
         private val NOT_RUNTIME_EDITABLE_ARRAY = arrayOf(
             MAIN_CUSTOM_RTC_VALUE,
-            MAIN_GFX_BACKEND
+            MAIN_GFX_BACKEND,
+            GFX_DRIVER_LIB_NAME
```
Comment (JosJuice) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/model/StringSetting.kt:105]: Are you sure this actually works? There's no setting in SettingsFragmentPresenter that's directly mapped to `GFX_DRIVER_LIB_NAME`. Rather, there's an option to open a dialog, and what's inside that dialog is custom and doesn't take `NOT_RUNTIME_EDITABLE_ARRAY` into account.

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/SettingsFragmentPresenter.kt [hunk 2]
```diff
@@ -1265,7 +1265,7 @@ class SettingsFragmentPresenter(
             )
         )
 
-        if (GpuDriverHelper.supportsCustomDriverLoading() && this.gpuDriver != null) {
+        if (GpuDriverHelper.supportsCustomDriverLoading() && this.gpuDriver != null && this.gameId.isNullOrEmpty()) {
```
Comment (t895) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/SettingsFragmentPresenter.kt:1268]:
```suggestion
        if (GpuDriverHelper.supportsCustomDriverLoading() && this.gpuDriver != null && this.gameId.isNullOrEmpty() && !NativeLibrary.IsRunning()) {
```
Do this and remove the item you put in the StringSetting array

**Comment by K0bin** [comment]
Done @t895

**Review by JosJuice — APPROVED** [review]

**Pull request merged**